ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/property.proto

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package com.daml.ledger.api.v1;
+
+
+option java_outer_classname = "PropertyOuterClass";
+option java_package = "com.daml.ledger.api.v1";
+option csharp_namespace = "Com.Daml.Ledger.Api.V1";
+
+message Property {
+  // Required
+  // We follow K8s' convention:
+  // '''
+  // The name segment is required and must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between.
+  // '''
+  // https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/#syntax-and-character-set
+  string name = 1;
+  // Optional
+  // At most 1024 bytes of information
+  bytes value = 2;
```

Comment:
Values are for the LAPI client to interpret so it should be fine to make them even more opaque (string -> bytes)

By using raw bytes instead of strings, on the participant side, we don't have to care about character encodings or the exact length in bytes of the stored data.

Reply:
I'd suggest to keep them as strings, so that it is easy for humans to interpret them; and also because this follows the k8s design that we want to copy to inherint both its qualities that are known to use and the ones that are not (yet) known to us.

I'd suggest we go for the the same limit as k8s: https://stackoverflow.com/questions/68268493/what-is-the-syntax-of-annotation-values-in-kubernetes#:~:text=The%20only%20validation%20applied%20to,t%20be%20longer%20than%20256Kb.

Reply:
For the record: discussed offline and settled on using `map<string, string> annotations = 12;`